src/CMakeLists.txt

```diff
@@ -92,6 +92,12 @@ add_third_party(
   LIB "none"
 )
 
+add_third_party(
+  unialgo
+  URL https://github.com/uni-algo/uni-algo/archive/refs/tags/v1.0.0.tar.gz
+  LIB "none"
```

Comment:
do you prefer header only? this has potential to bloat an object code, no?
it is possible to build a static lib too.